### DISCUSSION TITLE
Настройка на OpenWrt

### DISCUSSION THREAD

**Discussion by egorovna-developments** [xtrime-ru/antizapret-vpn-docker#31] [Q&A] [answered]
Помогите настроить на роутере под openwrt. Раньше всё было настроено по инструкции оригинального antizapret. Пересоздал контейнер с нуля со всеми изменениями и поменял файл настроек на роутере. Перестало открывать вообще что либо. Никуда не пускает.

**Comment by egorovna-developments** [comment]
Видел в другом вопросе по youtube что должно попадать в подсеть 10.224.0.0, но у меня уходит всё в 10.224.0.1

**Comment by egorovna-developments** [comment]
Вроде разобрался. Нашёл в pull requests доку на openwrt. Помогла простая замена в конфиге "dev tun" на "dev tun0". Интерфейс конкретно в роутере так назван.

**Reply by xtrime-ru** [reply, marked as answer]
Cпасибо @S-trace за доку!
На всякий случай оставлю ссылку на PR: 
https://github.com/xtrime-ru/antizapret-vpn-docker/pull/19/files
